rust/agama-server/tests/service.rs

```diff
@@ -77,8 +78,8 @@ async fn test_access_protected_route() -> Result<(), Box<dyn Error>> {
 // TODO: The following test should belong to `auth.rs`.
```

Comment:
is this TODO still valid?

Reply:
Yes, it is still valid. It is unrelated to this change.